code/API_definitions/README.md

```diff
@@ -0,0 +1,13 @@
+The Edge Cloud suite of APIs is split into 4 categories, based on the intents listed [here](https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/Harmonisation%20of%20APIs/describing%20and%20harmonising%20the%20Edge%20APIs.md).
+
+# Discovery
+Discover the Edge site with the shortest network path to an end user terminal,find out which is the optimal Edge site for your specifc application, register your Edge-hosted application endpoints for rapid connectivity to the optimal Edge site.
+
+Workload Orchestration
```

Comment:
Apart form this small change, I think we can merge

```suggestion
# Workload Orchestration
```

Reply:
Should we replace "Edge site" with Edge Cloud Zone?

Reply:
As well.